packages/express-session/src/index.ts

```diff
@@ -0,0 +1,119 @@
+import { AccountsServer } from '@accounts/server';
+import { Tokens, User } from '@accounts/types';
+import { Request, Response, NextFunction } from 'express';
+import * as requestIp from 'request-ip';
+import 'express-session';
+import { getUserAgent } from './utils/get-user-agent';
+
+export interface AccountsSessionOptions {
+  user?: {
+    name?: string;
+    resolve?: (tokens: Tokens) => User | Promise<User>;
+  };
+  name?: string;
+}
+
+export class AccountsSession {
+  private options: AccountsSessionOptions;
+
+  constructor(private accountsServer: AccountsServer, options: AccountsSessionOptions) {
+    this.options = {
+      name: 'accounts-js-tokens',
+      user: {
+        name: 'user',
+        resolve: async tokens => {
+          const session = await this.accountsServer.findSessionByAccessToken(tokens.accessToken);
+
```

Comment:
I think the default behavior should be to resume the session that will check the tokens are not expired, you can take a look at this file https://github.com/accounts-js/accounts/blob/master/packages/rest-express/src/user-loader.ts

Reply:
Done